packages/stable/src/__tests__/api/documentsApi.int.spec.ts

```diff
@@ -3,12 +3,55 @@
 import CogniteClient from '../../cogniteClient';
 import { setupLoggedInClient } from '../testUtils';
 import { DocumentSearchResponse } from '@cognite/sdk-stable/dist';
+import { TextEncoder } from 'util';
+
+const getFileId = async (
+  client: CogniteClient,
+  deadline: number = 15000
+): Promise<number> => {
+  try {
+    const resp = await client.documents.list({
+      limit: 1,
+    });
+    return resp.items[0].id;
+  } catch (error) {
```

Comment:
I think it's unexpected to get an exception for an empty result. Is this because you get an exception for index out of bounds in the list lookup? I think it would be better to check the length instead.